docs/data/base/getting-started/quickstart/quickstart.md

```diff
@@ -0,0 +1,244 @@
+# Quickstart
+
+<p class="description">Get started with Base UI, a library of headless ("unstyled") React UI components and low-level hooks.</p>
+
+## Installation
+
+`@mui/base` is completely standalone – run one of the following commands to add Base UI to your React project:
+
+### With npm
+
+```sh
+npm install @mui/base
+```
+
+### With yarn
+
+```sh
+yarn add @mui/base
+```
+
+### With pnpm
+
+```sh
+pnpm add @mui/base
+```
+
+### Peer dependencies
+
+<!-- #react-peer-version -->
+
+[`react`](https://www.npmjs.com/package/react) >= 17.0.0 and [`react-dom`](https://www.npmjs.com/package/react-dom) >= 17.0.0 are peer dependencies.
+
+## Implementing a Button
+
+This is a quick tutorial that goes through the basics of using and styling Base components by replicating a button from Github's UI, using their [style guide](https://primer.style) as a reference.
+
+{{"demo": "Tutorial.js", "defaultCodeOpen": false, "hideToolbar": true}}
+
+### Components & Hooks
+
+Base UI provides a ButtonUnstyled component and a useButton hook, both of which can be used to build a button, each with its own [benefits and trade-offs](/base/getting-started/usage/#components-vs-hooks).
+
+#### ButtonUnstyled Component
+
+```tsx
+import * as React from 'react';
+import ButtonUnstyled from '@mui/base/ButtonUnstyled';
+
+export default function App() {
+  return (
+    <div>
+      <ButtonUnstyled>ButtonUnstyled</ButtonUnstyled>
+    </div>
+  );
+}
+```
+
+#### useButton Hook
+
+```tsx
+import * as React from 'react';
+import useButton from '@mui/base/useButton';
+
+export default function App() {
+  const buttonRef = React.useRef();
+
+  const { getRootProps } = useButton({
+    ref: buttonRef,
```

Comment:
There's no need to pass a ref if you don't do anything with it. `useButton({})` should be enough in this case (but I just realized it looks weird and we should allow `useButton()`)

+ other places that show to use a hook

Reply:
👌 Got it, I've opened some related issues:
- https://github.com/mui/material-ui/issues/36880
- https://github.com/mui/material-ui/issues/36879